fixchain/ratelimiter/limiter_test.go

```diff
@@ -42,13 +43,15 @@ func TestRateLimiterSingleThreaded(t *testing.T) {
 			if qps > float64(limit)*1.01 {
 				t.Errorf("#%d: Too many operations per second. Expected ~%d, got %f", i, limit, qps)
 			}
+			log.Infof("#%d: Expected ~%d, got %f", i, limit, qps)
```

Comment:
I think this is redundant. The expected value is part of the test name, and the `qps` only needs printing if it's not matching the expected value, which already happens 2 lines above.

Same below.